src/frontends/pytorch/src/op/expand.cpp

```diff
@@ -46,23 +48,38 @@ OutputVector translate_expand_fx(const NodeContext& context) {
     num_inputs_check(context, 2, num_inputs);
     auto x = context.get_input(0);
     std::vector<int32_t> shape_vec;
-    auto sizes = context.get_input(1);
-    if (num_inputs != 2) {
+    if (context.get_input_type(1).is<type::List>()) {
```

Comment:
Can't you use `concat_dims_to_shape` here too?

Reply:
Updated accordingly